docs/content/core/system-ssh-agent.md

```diff
@@ -73,7 +63,27 @@ SECRET_SSH_KEY_ACQUIA='acquia_key'
 
 When creating the variable use the file name within `$HOME/.ssh/` directory as the variable value. NOTE: the **private** key should be referenced, not the public one.
 
+
 ## Manually Add Keys
 
 To add a key manually, run `fin ssh-key add <key_filename>` and it will be manually loaded. NOTE: this key will only be 
-be available to the containers until the next docksal-ssh-agent restart.
+available to the containers until the next ssh-agent restart.
+
+
+## Using the host's SSH agent
+
+The default behavior for the ssh-agent service is to manage SSH keys itself. There are two ways to enable SSH agent
+pass-through:
+
+  * When running Docksal as part of a CI job, set the environment variable `CI` to `1`. This will result in ssh-agent
+    starting in proxy mode by default.
+  * By setting the environment variable `DOCKSAL_SSH_AGENT_USE_HOST` to `1`. This will explictly enable proxy mode. Set
+    it by running `fin config set --global DOCKSAL_SSH_AGENT_USE_HOST=1` followed by `fin system reset ssh-agent`.
+
+On Linux, ssh-agent is able to directly mount your host SSH agent's Unix socket into the container.
+
+On Windows and macOS, the `socat` utility is used to convert your local Unix socket to a TCP-endpoint and back again.
+To do so, a local port configured by the `DOCKSAL_SSH_PROXY_PORT` environment variable (defaults to `30001`) is opened.
+
+**Caution:** if other users have local access to your system, they could connect to this port and use your keys. The
+port is only opened on an internal network interface and is not accessible from e.g. your LAN. Use with care!
```

Comment:
Are you using “e.g.” to mean “for example,” or is it a definitive statement? It’s placement here is awkward.

Reply:
It's meant to be read as "for example". Do you have a suggestion for a better sentence structure? My goal is to differentiate between local and external network access.

Reply:
I think just removing “e.g.” here would make it read better.

Reply:
Updated the statement.